app/assets/javascripts/textlab/views/attribute-modal-dialog.js

```diff
@@ -94,44 +94,44 @@ TextLab.AttributeModalDialog = Backbone.View.extend({
       var zoneLabelPrefix = this.model.getZoneLabelPrefix();
       var defaultZone = this.currentZone ? zoneLabelPrefix + this.currentZone.get('zone_label') : '';
 
-      return this.partials.dropdownInput( { 
-        field_name: this.fieldIDTemplate({ index: elementIndex, key: key }), 
-        field_title: attribute.displayName, 
-        field_value: defaultZone, 
-        field_instructions: attribute.instructions, 
+      return this.partials.dropdownInput( {
+        field_name: this.fieldIDTemplate({ index: elementIndex, key: key }),
+        field_title: attribute.displayName,
+        field_value: defaultZone,
+        field_instructions: attribute.instructions,
         no_blank: true,
         options: zones,
-        error: false 
-      }); 
-     } 
+        error: false
+      });
+     }
     else {
-      return this.partials[ attribute.fieldType + 'Input' ]( { 
-        field_name: this.fieldIDTemplate({ index: elementIndex, key: key }), 
-        field_title: attribute.displayName, 
-        field_value: attribute.defaultValue ? attribute.defaultValue : '', 
-        field_instructions: attribute.instructions, 
+      return this.partials[ attribute.fieldType + 'Input' ]( {
+        field_name: this.fieldIDTemplate({ index: elementIndex, key: key }),
+        field_title: attribute.displayName,
+        field_value: attribute.defaultValue ? attribute.defaultValue : '',
+        field_instructions: attribute.instructions,
         options: (typeof attribute.vocab === "string") ? this.config.vocabs[attribute.vocab] : attribute.vocab,
-        error: false 
-      }); 
+        error: false
+      });
     }
   },
 
   render: function() {
-    
+
     // prepare list of options for zone drop down
     var zoneLabelPrefix = this.model.getZoneLabelPrefix();
-    var zoneOptions = _.map( this.model.zones.models, function( zone ) {
+    var zoneOptions = _.map((this.model.zones || {}).models, function( zone ) {
```

Comment:
This is the only real change. Everything else is automatic whitespace correction from the IDE I use.

Reply:
Great, thanks for the quick fix!